addons/im_livechat/models/res_partner.py

```diff
@@ -38,3 +38,14 @@ def search_for_channel_invite(self, search_term, channel_id=None, limit=30):
     def _compute_user_livechat_username(self):
         for partner in self:
             partner.user_livechat_username = next(iter(partner.user_ids.mapped('livechat_username')), False)
+
+    def mail_partner_format(self, fields=None):
+        partner_format = super().mail_partner_format(fields=fields)
+        if not fields:
+            fields = {'user_livechat_username': True}
```

Comment:
I would not always add this by default, it's a lot of wasted bytes.
I'd prefer to always explicit it when we need it.